features/facebookImport/src/components/activitiesMinistory/activitiesMinistory.jsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState } from "react";
+
+import i18n from "../../i18n";
+
+import BarChartHorizontal from "../dataViz/barChartHorizontal.jsx";
+
+import { fillArray } from "../dataViz/utils.jsx";
+
+import "./activitiesMinistory.css";
+import "./datePicker.css";
+import "../tabs/tabs.css";
+
+const monthsAbbreviation = [
```

Comment:
Small issue, but these could also be different depending on the language Oct/Okt; Mar/Mär